### PR TITLE
Fix offenses from the new version of rubocop.

### PR DESCRIPTION
**Pull request by dylanahsmith**
@fw42 for review

I noticed that CI is now picking up new rubocop offenses, since we don't lock the version in the Gemfile.  Should we be locking the version?

Either way, this should deal with these new offenses.

### CODE REVIEW

File: lib/liquid/standardfilters.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def escape_once(input)
     end
 
     def url_encode(input)
-      CGI.escape(input) rescue input
```
Comment (dylanahsmith) [inline lib/liquid/standardfilters.rb]: We already removed the `rescue input` for the escape filter in https://github.com/Shopify/liquid/pull/611, so we may as well do it here as well for the same reasons.

**Comment by fw42** [comment]
Locking the version makes sense!

**Comment by fw42** [comment]
:+1:

**Comment by dylanahsmith** [comment]
Ok.  I've locked the version to the latest version of rubocop.

**Pull request merged**